### DISCUSSION TITLE
Doris real-time data warehouse tiered storage

### DISCUSSION THREAD

**Discussion by 15374604475** [apache/doris#19766] [RFC]
想用doris做实时数仓分层，dwd层->dws层做实时etl有什么好的办法吗？除了在doris层用insert into，这样会损耗doris性能，并且doris的定位并不是用来做etl工具，那请问还有其他办法吗？

**Comment by LOVEGISER** [comment]
1. 对于亿级别数据，ETL用Doris还是没有压力的，并且性能比Spark和Hive会快很多
2. 如果担心ETL影响集群稳定性，可以使用Doris资源组隔离方案。
3. 除了inesrt into select...  还可使用物化视图和rollup，后面会支持多表物化视图
4. Doris 海量数据ETL过程数据spill到磁盘已经研发的差不多了，很快会推出。届时海量数据ETL（超过集群内存的数据ETL）将变为可能。

**Reply by RyomaXu** [reply]
今年2.0版本支持海量数据的ETL吗TB级别

**Reply by zlatangao** [reply]
你好，请教一下，可以帮忙讲讲，是如何“使用doris实现etl的功能”的吗？🙏

**Comment by RyomaXu** [comment]
您好！您的邮件我已经收到，谢谢！
